Examples/HttpAnalyzer/HttpStatsCollector.h

```diff
@@ -465,4 +465,5 @@ class HttpStatsCollector
 
 	double m_LastCalcRateTime;
 	double m_StartTime;
+	unsigned short m_dstPort;

```

Comment:
instead of `unsigned short` you can use `uint16_t` which is the convention in the rest of the library